js/main.js

```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line no-unused-vars
+function changeIcon(icon) {
+  icon.classList.toggle('fa-times');
+}
+
+const navLinks = document.querySelectorAll('.nav-item');
+const menuToggle = document.getElementById('collapsibleNavbar');
+// eslint-disable-next-line no-undef
```

Comment:
[OPTIONAL] Please don't disable linters. They are important for code standardization for collaborative projects. ⚠️